.github/workflows/trigger-ballerinax-connectors-build.yml

```diff
@@ -3,7 +3,7 @@ name: Trigger ballerinax connectors build
 on:
   workflow_dispatch:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '30 18 * * *'   # 00:00 in LK time (GMT+5:30)
```

Comment:
This should be run after the distribution daily build is run. Shall we check when distribution daily build will be run?